content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Creating Projects using the Web Editor'
+description: 'An article on how a teacher can use the web editor '
+tags: 
+  - article
+author: 'Lenard George Swamy'
+---
+
+## Introduction 
+
+Many of you might have an interesting idea for your project but struggle with documentation and sharing it with your class. In this article you will learn how to use the web editor to document a project. We will illustrate the process with an example project  
+
+***Note: We recommend that you read the article (Getting started with the Web Editor)(/getting-started/getting-started-web-editor) before you proceed with this tutorial.***
+
+## The Essentials 
+Every arduino project has 3 essential elements that will make your teaching effortless. These are
+* Circuit diagrams 
+* Sketch Files 
+* Picture of your project 
+
+### 1. Create a circuit diagram
+Start by thinking about the layout of your circuit. It could be as basic as a pen on paper but more the fidelity the easier it is for your students to understand how to connect the components. 
+
+For this tutorial we created a circuit of the proximity meter using [tinkercad](https://www.tinkercad.com/circuits).
+
+![proximity_meter_circuit.png](assets/classroom_project_circuit_diagram_proximity_meter.png)
+
+### 2. Connect the components 
+Once you have created the components, its good to connect the components to ensure all the connections are right. Sometimes its also good to support your document with photos of your connections. Heres a picture of the circuit. This gives your students a good visual sense of how the circuit looks like in real life.
+
+![proximity_meter_real_components.png](assets/classroom_project_photo_proximity_meter.png)
+
+### 3. Open a new sketch.
+Open the Web Editor by clicking on the ![header_dropdown_icon.png](assets/icon_header_dropdown.png) icon in the header. 
+
+![proximity_meter_real_components.png](assets/classroom_project_webeditor.png)
+
+When your editor is ready, create a new sketch by clicking ![new_sketch_icon.png](assets/icon_new_sketch.png) and rename the sketch to the name of your project. 
```

Comment:
same here - the icon doesn't show up "in the text" but rather on a new line and it breaks the  structure of the tutorial